tools/releases/dockerfiles/Dockerfile.kuma-init

```diff
@@ -15,6 +22,11 @@ COPY /tools/releases/templates/LICENSE \
 
 COPY /tools/releases/templates/NOTICE-kumactl /kuma/NOTICE
 
+# ebpf
+COPY --from=merbridge /app/bpf/*.o /kuma/ebpf/
+COPY --from=merbridge /usr/local/sbin/bpftool /usr/bin/bpftool
+COPY /tools/builds/ebpf/load-and-attach.mk /kuma/ebpf/Makefile
```

Comment:
Anyway to have this build done in either merbridge or kuma-net?

Reply:
I will think about how to do it

Reply:
Done: https://github.com/kumahq/kuma-net/pull/74